.eslintignore

```diff
@@ -0,0 +1,17 @@
+dashboard/public/js/bootstrap-notify.js
+dashboard/public/js/bootstrap.min.js
+dashboard/public/js/Chart.min.js
+dashboard/public/js/jquery-3.1.0.min.js
+dashboard/public/js/jquery-sortable.js
+dashboard/public/js/jquery-ui-1-11-4.js
+dashboard/public/js/jquery.multi-select.js
+dashboard/public/js/jquery.searchable.js
```

Comment:
`cant you just use dashboard/public/*`

Reply:
We need to lint files such as `main.js` which is stored in the same directory.

P.S. I guess we can further restructure the project to separate public resources with project js files

Reply:
yeah you can format this file https://github.com/llaske/sugarizer-server/blob/master/dashboard/public/js/main.js and move code to respective controllers.